src/StateManager.ts

```diff
@@ -24,11 +24,11 @@ export class StateManager {
   static clearState() {
     this.workspaceState.keys().forEach((key) => {
       if (
-          key !== KEYS.PROJECT_ID &&
-          key !== KEYS.ORGANIZATION &&
-          key !== KEYS.AUTH0_USER_ID &&
-          key !== KEYS.SEND_METRICS_PROMPTED &&
-          key !== KEYS.EXTENSION_INSTALLED
+          !(key.includes(KEYS.PROJECT_ID)||
+          key.includes(KEYS.ORGANIZATION)||
+          key.includes(KEYS.AUTH0_USER_ID)||
+          key.includes(KEYS.SEND_METRICS_PROMPTED)||
+          key.includes(KEYS.EXTENSION_INSTALLED))
```

Comment:
`EXTENSION_INSTALLED` is stored in global state so we don't need to check it. Also I think `SEND_METRICS_PROMPTED` belongs in global state as well but we can fix that in another pr